va/va_trace.c

```diff
@@ -6316,3 +6316,45 @@ void va_TraceEventBuffers(
     }
     return;
 }
+
+void va_TraceExportSurfaceHandle(
+    VADisplay        dpy,
+    VASurfaceID      surfaceId,
+    uint32_t         memType,
+    uint32_t         flags,
+    void             *descriptor)
+{
+    int i;
+
+    DPY2TRACE_VIRCTX(dpy);
+
+    TRACE_FUNCNAME(idx);
+
+    va_TraceMsg(trace_ctx, "\tsurfaceId = 0x%08x\n", surfaceId);
+    va_TraceMsg(trace_ctx, "\tmemType   = 0x%08x\n", memType);
+    va_TraceMsg(trace_ctx, "\tflags     = 0x%08x\n", flags);
+
+    VADRMPRIMESurfaceDescriptor *desc = (VADRMPRIMESurfaceDescriptor *)descriptor;
+
+    va_TraceMsg(trace_ctx, "\tfourcc      = %u\n", desc->fourcc);
```

Comment:
This will crash if descriptor is NULL. Parent function (`vaExportSurfaceHandle`) does not check that (and it should not). Add check here, please.

Reply:
Fixed in last version, see line 6343 above.